internal/pkg/cli/env_delete.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// DeleteEnvOpts holds the fields needed to delete an environment.
+type DeleteEnvOpts struct {
+	Env              string
+	SkipConfirmation bool
+}
+
+// Ask is a no-op for this command.
+func (opts *DeleteEnvOpts) Ask() error {
```

Comment:
Except the confirmation.

Reply:
I think the confirmation will be in `Execute`